File: dsa in python/linear search.py
arr=list(range(1, 21))
target = int(input("Enter target you want to find: "))
for j in range(len(arr)-1):
    if arr[j]==target:
        print(target,"found at index",j)
    else:
        print(target,"not found in array 1-20")

File: dsa in c/linearsearch.c
#include <stdio.h>
int main(){
    int N = 20, arr[N], target;
    printf("Enter %d numbers: ", N);
    for(int i = 0; i < N; i++) scanf("%d", &arr[i]);
    printf("Enter number to find: ");
    scanf("%d", &target);
    for(int j = 0; j < N; j++){
        if(arr[j] == target){
            printf("%d found at index %d.\n", target, j);
            return 0;
        }
    }
    printf("%d not found in given array.\n", target);

    return 0;
}
